src/core/constraint_template.jl

```diff
@@ -182,58 +182,58 @@ function constraint_tp_flow_losses(pm::_PMs.GenericPowerModel, i::Int; nw::Int=p
     b_sh_fr = branch["b_fr"].values
     b_sh_to = branch["b_to"].values
 
-    constraint_tp_flow_losses(pm::_PMs.GenericPowerModel, nw, i, f_bus, t_bus, f_idx, t_idx, r, x, g_sh_fr, g_sh_to, b_sh_fr, b_sh_to)
+    constraint_mc_flow_losses(pm::_PMs.AbstractPowerModel, nw, i, f_bus, t_bus, f_idx, t_idx, r, x, g_sh_fr, g_sh_to, b_sh_fr, b_sh_to)
 end
 
 
 ""
-function constraint_tp_storage_exchange(pm::_PMs.GenericPowerModel, i::Int; nw::Int=pm.cnw)
+function constraint_mc_storage_exchange(pm::_PMs.AbstractPowerModel, i::Int; nw::Int=pm.cnw)
```

Comment:
Following PowerModels conventions, `storage_exchange` has been replaced by separate constraint templates for `constraint_storage_complementarity_nl` and `constraint_storage_loss`.  This can be posted as an issue to fix later if it is time consuming to fix here.

We should post an issue,